schema.sql

```diff
@@ -0,0 +1,40 @@
+CREATE DATABASE IF NOT EXISTS books;
+
+use books;
+
+CREATE TABLE IF NOT EXISTS details (
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  type VARCHAR(20),
+  pagenum INT,
+  publisher VARCHAR(100),
+  firstPubDate VARCHAR(30),
+  originalPubDate VARCHAR(30),
+  title VARCHAR(100),
+  isbn10 VARCHAR(20),
+  isbn13 VARCHAR(20),
```

Comment:
information about such as publisher, originalPubDate, isbn10, isbn13, etc looks specific to a particular book edition and they have been reflected in the edition table, hence repeating them in the details table looks redundant.

Reply:
Each book has a main edition data that is refered to in the details table. All other editions of the book are considered secondary and refered to in the editions table. For goodreads.com the main edition of the book is selected due to a number of factor such as, where the user is accessing the site from and how recently each edition was printed or how popular each edition was. For example, a book that was first published 100 years ago would not show the first published edition as its main edition. It would show the edition that is most readily available for purchase at stores currently.